src/phoenix/db/migrate.py

```diff
@@ -25,4 +47,7 @@ def migrate(url: URL) -> None:
     alembic_cfg.set_main_option("script_location", scripts_location)
     alembic_cfg.set_main_option("sqlalchemy.url", str(url))
 
-    command.upgrade(alembic_cfg, "head")
+    with using_log_level(logging.DEBUG, ["sqlalchemy", "alembic"]):
```

Comment:
It would be helpful to have this as an environment variable.

Reply:
Agreed, we need a pythonic way to do something like https://www.npmjs.com/package/debug